pkg/lib/dialogs.jsx

```diff
@@ -106,7 +106,14 @@ export const WithDialogs = ({ children }) => {
     const [dialog, setDialog] = useState(null);
 
     const Dialogs = {
-        show: setDialog,
+        show: component => {
+            if (component && dialog !== null)
+                console.error("Dialogs.show() called for",
+                              JSON.stringify(component),
+                              "while a dialog is already open:",
+                              JSON.stringify(dialog));
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19595-20231110-083247-9c91eb53-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)